inc/functions.php

```diff
@@ -85,17 +85,17 @@ function core_sitemaps_get_sitemaps() {
  * @since 5.5.0
  *
  * @param string                 $name     Unique name for the sitemap provider.
- * @param Core_Sitemaps_Provider $provider The `Core_Sitemaps_Provider` instance implementing the sitemap.
+ * @param Sitemaps_Provider $provider The `Sitemaps_Provider` instance implementing the sitemap.
  * @return bool Returns true if the sitemap was added. False on failure.
  */
-function core_sitemaps_register_sitemap( $name, $provider ) {
-	$core_sitemaps = core_sitemaps_get_server();
+function sitemaps_register_sitemap( $name, $provider ) {
```

Comment:
`register_sitemap`?

Reply:
👍

Reply:
going with `wp_register_sitemap` for consistency with `wp_get_sitemaps` If we decide to remove the wp_ prefix later, they will be easy to find.

Reply:
Should we rename the core class from `Sitemaps` to `WP_Sitemaps`?

Reply:
I wound up adding a `wp_` prefix for functions in the main functions.php file, see https://github.com/GoogleChromeLabs/wp-sitemaps/pull/182/commits/42f663b47946c19ce03ccd8af19179af73f532c8